packages/wrangler/src/index.tsx

```diff
@@ -99,6 +99,9 @@ const proxy =
 
 if (proxy) {
 	setGlobalDispatcher(new ProxyAgent(proxy));
+	logger.log(
+		`Proxy environment variables detected. Using ${proxy} for all fetch requests.`
```

Comment:
Question is: do we think `proxy` would contain sensitive information? 

in that case we shouldn't log the address...